tools/integration_tests/gzip/write_gzip_test.go

```diff
@@ -50,12 +50,17 @@ func verifyFullFileOverwrite(t *testing.T, filename string) {
 			mountedFilePath, (*fi).Size(), gcsObjectPath, gcsObjectSize)
 	}
 
+	content, err := helpers.CreateDataOfSize(OverwittenFileSize)
```

Comment:
Pre-existing typo. OverwittenFileSize . Please fix.

```suggestion
	content, err := helpers.CreateDataOfSize(OverwrittenFileSize)
```